src/App.tsx

```diff
@@ -1,24 +1,227 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import { useState, useEffect, useMemo } from 'react';
+import { Header } from './components/Header/Header';
+import { Footer } from './components/Footer/Footer';
+import { TodoList } from './components/TodoList/TodoList';
+import { ErrorNotification }
+  from './components/ErrorNotification/ErrorNotification';
 
-const USER_ID = 0;
+import {
+  getTodos, addTodo, deleteTodo, updateStatusTodo, updateTitleTodo,
+} from './api/todos';
+import { getVisibleTodos } from './utils/getVisibleTodos';
+
+import { Todo } from './types/Todo';
+import { SortType } from './types/SortType';
+
+const USER_ID = 6650;
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const [todosList, setTodosList] = useState<Todo[]>([]);
+  const [query, setQuery] = useState('');
+  const [errorType, setErrorType] = useState('');
+  const [sortType, setSortType] = useState<SortType>(SortType.ALL);
+  const [isDisabledForm, setIsDisabledForm] = useState(false);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [tempTodos, setTempTodos] = useState<Todo[]>([]);
+  const [updatingTodo, setUpdatingTodo] = useState<Todo | null>(null);
+
+  const copletedTodos = todosList.filter(todo => todo.completed);
+  const activeTodos = todosList.filter(todo => !todo.completed);
+  const totalTodoListLength = todosList.length;
+  const activeTodoListLength = activeTodos.length;
+  const isActiveFooter = !!totalTodoListLength;
+  const completedTodoListLength = totalTodoListLength - activeTodoListLength;
+
+  const visibleTodoList = useMemo(
+    () => getVisibleTodos(todosList, sortType),
+    [todosList, sortType],
+  );
+
+  // Show Error Message
```

Comment:
comments should be in the line right above the code it related to
but in this case (and other cases in the project) code is clear enough without such comments - you may remove them

Reply:
fixed